app/jobs/submit_patron_request_job.rb

```diff
@@ -63,17 +63,17 @@ def illiad_recall_statuses
   end
 
   def convert_to_mediated_page(patron_request)
-    MediatedPage.create!(
+    page = MediatedPage.create!(
       needed_date: patron_request.needed_date,
-      origin: patron_request.origin_library_code,
+      origin: patron_request.origin_library_code, origin_location: patron_request.origin_location_code,
```

Comment:
Why did this get moved to a new line? Kind of nice everything is on single lines for everything else.

Reply:
Rubocop didn't like the method length 🤷‍♂️